marconi-chain-index/src/Marconi/ChainIndex/Utils.hs

```diff
@@ -64,3 +66,7 @@ chainPointOrGenesis :: [C.ChainPoint] -> C.ChainPoint
 chainPointOrGenesis result = case result of
   []     -> C.ChainPointAtGenesis
   cp : _ -> cp
+
+-- | Convert a list of target addresses to a predicate function
+addressesToPredicate :: Maybe TargetAddresses -> Maybe (C.Address C.ShelleyAddr -> Bool)
```

Comment:
That should maybe go in the Utxo indexer? That's the only place we're using it.

Reply:
It's was used in the AddressDatum indexer, the move out was to re-use it in the Utxo indexer. (Maybe all the refactor looks better after it's closer to complete, as now it was half way there and looks funny as it's only used in one single other place.)